api/src/main/java/org/reactivestreams/Publisher.java

```diff
@@ -15,7 +15,7 @@
  * A {@link Publisher} is a provider of a potentially unbounded number of sequenced elements, publishing them according to
  * the demand received from its {@link Subscriber}(s).
  * <p>
- * A {@link Publisher} can serve multiple {@link Subscriber}s subscribed {@link #subscribe(Subscriber)} dynamically
+ * A {@link Publisher} can serve multiple {@link Subscriber}(s) subscribed {@link Publisher#subscribe(Subscriber)} dynamically
```

Comment:
`multiple` already indicates plural here so `s` is fine.

Reply:
Oops missed that, good catch.